### ISSUE TITLE
Received unknown parameter: default_tax_rates

### ISSUE THREAD

**Issue by dalholm** [laravel/cashier-stripe#1049]
- Cashier Version: 12.7.0
- Laravel Version: 8.2.0
- PHP Version: 8.0.0

### Description:
I do not know if I am doing anything wrong, but I can not get tax to work on subscriptions (checkout) without changing the package.

With the original code i get this error: `Received unknown parameter: default_tax_rates`

But if i follow [stripe docs](https://stripe.com/docs/billing/subscriptions/taxes#adding-tax-rates-to-checkout) and change the package to include the `default_tax_rates` in `subscription_data` it works perfectly. 

```php

// SubscriptionBuilder.php

public function checkout(array $sessionOptions = [], array $customerOptions = [])
    {
        if (! $this->skipTrial && $this->trialExpires) {
            // Checkout Sessions are active for 24 hours after their creation and within that time frame the customer
            // can complete the payment at any time. Stripe requires the trial end at least 48 hours in the future
            // so that there is still at least a one day trial if your customer pays at the end of the 24 hours.
            $minimumTrialPeriod = Carbon::now()->addHours(48);

            $trialEnd = $this->trialExpires->gt($minimumTrialPeriod) ? $this->trialExpires : $minimumTrialPeriod;
        } else {
            $trialEnd = null;
        }


        return Checkout::create($this->owner, array_merge([
            'mode' => 'subscription',
            'line_items' => collect($this->items)->values()->all(),
            'allow_promotion_codes' => $this->allowPromotionCodes,
            'discounts' => [
                'coupon' => $this->coupon,
            ],
            // 'default_tax_rates' => $this->getTaxRatesForPayload(),  <----  Remove this, causing Received unknown parameter: default_tax_rates
            'subscription_data' => [
                'default_tax_rates' => $this->getTaxRatesForPayload(), // <---- add this referring to the docs https://stripe.com/docs/billing/subscriptions/taxes#adding-tax-rates-to-checkout
                'trial_end' => $trialEnd ? $trialEnd->getTimestamp() : null,
                'metadata' => array_merge($this->metadata, ['name' => $this->name]),
            ],
        ], $sessionOptions), $customerOptions);
    }
```

## Alternative solution

1. Remove `taxRates()` from your 'billable' model
2. Add following to your checkout creation

```php
$user->newSubscription()->checkout([
    'subscription_data' => [
                'default_tax_rates' => ['txr_.........'],
               'metadata' => [
                       'name' =>  'default',
                  ],
            ]
])
```

**Comment by driesvints** [comment]
Thanks for reporting this! I've sent in a fix.

**Comment by driesvints** [comment]
Released v12.7.1 with the fix.

**Comment by dalholm** [comment]
@driesvints that was quick! Thanks!
